cves/kernel/CVE-2021-30002.yml

```
@@ -55,7 +55,13 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This vulnerability is a memory lead in the method video_usercopy that is
+  used by users who have a web camera. When the IOTCTL (input/output control)
```

If the usage details aren't important, this could be shortened
```suggestion
  used by web cameras. When the IOTCTL (input/output control)
```